sanic/src/server.py

```diff
@@ -93,4 +100,55 @@ async def savevideo(request: Request) -> HTTPResponse:
         file.write(videofile.body)
         file.close()
     else:
-        return text("No label provided!", 400)
\ No newline at end of file
+        return text("No label provided!", 400)
```

Comment:
cringe at du stjæler min linje

Reply:
`if (CasperKjaerhus == Cringe) {
    print ("He is cringe")
`
OUTPUT:
He is cringe